roles/manage-repositories/tasks/disable_repositories.yml

```diff
@@ -0,0 +1,19 @@
+---
+# tasks file for manage-repositories
+- name: "Install Yum Utils"
```

Comment:
Should this just be `yum-utils` given thats the package name?

Reply:
Will update it to `Install yum-utils`.